hazelcast-integration/kubernetes/samples/external-client/README.md

```diff
@@ -0,0 +1,201 @@
+NAME              READY     STATUS    RESTARTS   AGE
+pod/hazelcast-0   1/1       Running   0          2m
+pod/hazelcast-1   1/1       Running   0          1m
+pod/hazelcast-2   1/1       Running   0          1m
+
+NAME                  TYPE           CLUSTER-IP      EXTERNAL-IP      PORT(S)          AGE
+service/hazelcast-0   LoadBalancer   10.19.241.253   35.188.83.111    5701:30597/TCP   2m
+service/hazelcast-1   LoadBalancer   10.19.251.243   35.192.168.46    5701:32718/TCP   2m
+service/hazelcast-2   LoadBalancer   10.19.254.0     35.193.248.247   5701:30267/TCP   2m
+service/kubernetes    ClusterIP      10.19.240.1     <none>           443/TCP          1h
+
+$ kubectl logs pod/hazelcast-2
+...
+Members {size:3, ver:3} [
+        Member [10.16.1.10]:5701 - abab30fe-5a45-484d-bad5-e60c252572ca
+        Member [10.16.2.7]:5701 - 9b948e91-0115-470f-850e-d5cbf2e3b0e1
+        Member [10.16.0.8]:5701 - e68ce431-4000-467b-92c6-0072b2601d60 this
+]
+...
+```
+
+## Configure Hazelcast Client outside Kubernetes
+
+When we have a working Hazelcast cluster deployed on Kubernetes, we can connect to it with an external Hazelcast Smart Client. You need first to fetch the credentials of the created Service Account and then use them to configure the client.
+
+### 5. Check Kubernetes Master IP
+
+To check the IP address of the Kubernetes Master, use the following command.
+
+```
+$ kubectl cluster-info
+Kubernetes master is running at https://35.226.182.228
+```
+
+### 6. Check Access Token and CA Certificate
+
+First, you need to find the name of the secret for the created Service Account.
+
+```
+$ kubectl get secret
+NAME                                 TYPE                                  DATA      AGE
+default-token-q9sp8                  kubernetes.io/service-account-token   3         2h
+hazelcast-service-account-token-6s94h   kubernetes.io/service-account-token   3         9m
+```
+
+Then, to fetch Access Token, use the following command.
+
+```
+$ kubectl get secret hazelcast-service-account-token-6s94h -o jsonpath={.data.token} | base64 -d
+eyJhbGciOiJSUzI1NiIsImtpZCI6IiJ9.eyJpc3MiOiJrdWJlcm5ldGVzL3NlcnZpY2VhY2NvdW50Iiwia3ViZXJuZXRlcy5pby9zZXJ2aWNlYWNjb3VudC9uYW1lc3BhY2UiOiJkZWZhdWx0Iiwia3ViZXJuZXRlcy5pby9zZXJ2aWNlYWNjb3VudC9zZWNyZXQubmFtZSI6InNhbXBsZS1zZXJ2aWNlLWFjY291bnQtdG9rZW4tNnM5NGgiLCJrdWJlcm5ldGVzLmlvL3NlcnZpY2VhY2NvdW50L3NlcnZpY2UtYWNjb3VudC5uYW1lIjoic2FtcGxlLXNlcnZpY2UtYWNjb3VudCIsImt1YmVybmV0ZXMuaW8vc2VydmljZWFjY291bnQvc2VydmljZS1hY2NvdW50LnVpZCI6IjI5OTI1NzBmLTI1NDQtMTFlOS1iNjg3LTQyMDEwYTgwMDI4YiIsInN1YiI6InN5c3RlbTpzZXJ2aWNlYWNjb3VudDpkZWZhdWx0OnNhbXBsZS1zZXJ2aWNlLWFjY291bnQifQ.o-j4e-ducrMmQc23xYDnPr6TIyzlAs3pLNAmGLqPe9Vq1mwsxOh3ujcVKR90HAdkfHIF_Sw66qC9hXIDvxfqN_rLXlOKbvTX3gjDrAnyY_93Y3MpmSBj8yR9yHMb4O29a9UIwN5F2_VoCsc0IGumScU_EhPYc9mvEXlwp2bATQOEU-SVAGYPqvVPs9h5wjWZ7WUQa_-RBLMF6KRc9EP2i3c7dPSRVL9ZQ6k6OyUUOVEaPa1tqIxP7vOgx9Tg2C1KmYF5RDrlzrWkhEcjd4BLTiYDKEyaoBff9RqdPYlPwu0YcEH-F7yU8tTDN74KX5jvah3amg_zTiXeNoe5ZFcVdg
+```
```

Comment:
I had an issue with token copy/paste. An output with a new line would be clearer.
`kubectl get secret hazelcast-service-account-token-rnbtx -o jsonpath={.data.token} | base64 -D | xargs echo`